skin/src/main/java/org/researchstack/skin/DataProvider.java

```diff
@@ -105,7 +106,7 @@ public static void init(DataProvider instance)
      * @return Observable of the result of the method, with {@link DataResponse#isSuccess()}
      * returning true if signIn was successful
      */
-    public abstract Observable<DataResponse> resendEmailVerification(Context context, String email);
```

Comment:
that's how it works in my application. I am using a server which is partially compatible with Sage and the email needs the password too. I can leave the old signature too.